pages/clusters/queue_metrics.md

```diff
@@ -0,0 +1,24 @@
+# Queue metrics
+Queue metrics show you the most important statistics to help you optimize your agent setup and
+monitor your cluster's performance. These statistics are updated on the page every 10 seconds.
+
+## Agents panel
+Agents Connected is the number of agents connected to this cluster queue. The circular chart
+represents the fraction of agents that are busy working on jobs compared to those that are idle and
+ready for a job. Hovering over the Agents Connected chart pops up the Agent Utilization panel which
+shows the same information as the circular chart but with the numerical percentages of agents in
+use vs idle.
```

Comment:
I think we can simplify this a little. I'm also using italics to draw attention to the key term and UI text.

```suggestion
_Agents Connected_ is the number of agents connected to the cluster queue. The circular chart represents the fraction of agents that are busy working on jobs compared to those that are idle and ready for a job. Hovering over the chart shows the _Agent Utilization_ panel, which displays the percentage values for each chart component.
```